### DISCUSSION TITLE
SEARCH method

### DISCUSSION THREAD

**Discussion by jorgebarrero** [directus/directus#11326] [Feature Requests]
This is my opinion about the SEARCH method... 

Is clean and easy to understand. But... most testing tools (postman and others..) do not include a SEARCH option, so is not easy to test.

It would be nice to create a plugin for Directus, so that it can have a Postman like tool

**Comment by rijkvanzanten** [comment]
Here's to hoping the spec makes it through, so all your favorite tools will also have a reason to start natively supporting it (https://datatracker.ietf.org/doc/draft-ietf-httpbis-safe-method-w-body/?include_text=1)

**Reply by rijkvanzanten** [reply]
> But... most testing tools (postman and others..) do not include a SEARCH option, so is not easy to test.

For what it's worth, this really depends on the tool you're using. Postman has had a way to send custom HTTP methods for years (https://blog.postman.com/custom-http-methods-more-flexibility-and-autonomy/) 🙂

**Comment by jorgebarrero** [comment]
Always learning... I will give it a try with directus search method

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
